templates/webapi/test/settings.html.ep

```diff
@@ -4,9 +4,22 @@
     </thead>
     <tbody>
         % my $s = $job->settings_hash;
+        % my @link_keys = split ",", app->config->{job_settings_ui}->{keys_to_render_as_links};
+        % my $links = join '|', @link_keys;
```

Comment:
Don't add logic like this to the template. Better prepare this once after reading the config files and not every time the route is served. (It should be possible to add an additional attribute to the app for this value.)